group_project_v2/mixins.py

```diff
@@ -160,22 +327,18 @@ def workgroup(self):
         workgroup = self._get_workgroup(self.project_api, self.user_id, self.course_id)
         return workgroup if workgroup else self.FALLBACK_WORKGROUP
 
-    @staticmethod
-    @memoize_with_expiration()
-    def _get_workgroup(project_api, user_id, course_id):
+    def _get_workgroup(self, project_api, user_id, course_id):
         """
         :rtype: WorkgroupDetails
```

Comment:
@jbzdak  OpenCraft member
jbzdak added a note 4 minutes ago
This was memoized and static, now it is not. Rationale is that:

* All API calls are memoized so this is not a timesaver,
* we need settings here --- and settings rely on settings service, which require this to be an instance method